src/xrpld/app/tx/applySteps.h

```diff
@@ -29,6 +29,13 @@ class Application;
 class STTx;
 class TxQ;
 
+struct TxApplyResult
```

Comment:
This name is ambiguous. We have `apply` from `apply.h` and `doApply` from `applySteps.h` which return `TxApplyResult`. And `applyTransaction`  transaction from `apply.h` which return `ApplyResult`

You should rename
```
ApplyResult - >  ApplyTransactionResult
TxApplyResult ->  ApplyResult
```
At least it will be closer to the function names